package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "redesigned-webpack",
+  "version": "1.0.0",
+  "description": "A webpack template for future use",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/menyagah/redesigned-webpack.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/menyagah/redesigned-webpack/issues"
+  },
+  "homepage": "https://github.com/menyagah/redesigned-webpack#readme",
+  "devDependencies": {
```

Comment:
- [ ] The _dervServer_ package is missing your application. Kindly install the same and add that in your **webpack** configuration to enable your application to be run on the server locally.

**devServer in webpack configuration file:**
![image](https://user-images.githubusercontent.com/12745474/134922749-d0016f56-002e-4415-8882-4eb3dcbf6551.png)